.pre-commit-config.yaml

```diff
@@ -0,0 +1,11 @@
+ci:
+  autofix_commit_msg: "style: [pre-commit.ci] autofix"
+  autoupdate_commit_msg: "chore(deps): [pre-commit.ci] autoupdate"
```

Comment:
Do you need to link the repo to https://pre-commit.ci/ for these `ci` changes to take effect?

Reply:
Yes, I believe so. I do not have the privileges to sign-in there for this repository.